packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -528,13 +555,14 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
     }
     _floatingOptions?.remove();
     _floatingOptions = null;
+    WidgetsBinding.instance.removeObserver(this);
     super.dispose();
   }
 
   @override
   Widget build(BuildContext context) {
     return TextFieldTapRegion(
-      child: Container(
+      Container(
```

Comment:
It looks to me like this should be a compiler error since there is no `child:`, or am I misreading?

Reply:
My environment was badly setup, it is fixed